### DISCUSSION TITLE
How to tell if new split version is available?

### DISCUSSION THREAD

**Discussion by marsfan** [microsoft/winget-cli#3076] [Q&A] [answered]
Some packages are distributed in a "split version" form. For example, with Ruby, there are different packages for the different versions of Ruby: 

```
Name                Id                                   Version
-----------------------------------------------------------------
Ruby 3.1 with MSYS2 RubyInstallerTeam.RubyWithDevKit.3.1 3.1.2-1
Ruby 3.0 with MSYS2 RubyInstallerTeam.RubyWithDevKit.3.0 3.0.4-1
Ruby 2.7 with MSYS2 RubyInstallerTeam.RubyWithDevKit.2.7 2.7.6-1
Ruby 2.6 with MSYS2 RubyInstallerTeam.RubyWithDevKit.2.6 2.6.10-1
Ruby 3.1            RubyInstallerTeam.Ruby.3.1           3.1.2-1
Ruby 3.0            RubyInstallerTeam.Ruby.3.0           3.0.4-1
Ruby 2.7            RubyInstallerTeam.Ruby.2.7           2.7.6-1
Ruby 2.6            RubyInstallerTeam.Ruby.2.6           2.6.10-1
Ruby 3.2 with MSYS2 RubyInstallerTeam.RubyWithDevKit.3.2 3.2.0-1
```

If I have installed `RubyInstallerTeam.Ruby.3.1`,  is there a way for `winget` to tell me if `RubyInstallerTeam.Ruby.3.2` is available in case I want to upgrade to a new Ruby version?

**Comment by Trenly** [comment, marked as answer]
Currently, there isn't a way for this to happen. However, there is a feature request for this -
* https://github.com/microsoft/winget-cli/issues/1899

**Comment by MonroeSantaanafo** [comment]
To check if RubyInstallerTeam.Ruby.3.2 is available, you can run the command "winget show RubyInstallerTeam.Ruby.3.2".

**Reply by Masamune3210** [reply]
The whole point of the question was how to show possible upgrades for split manifest packages without manual intervention or searching.
